packages/schematics/angular/application/files/src/environments/environment.ts

```diff
@@ -1,7 +1,7 @@
-// The file contents for the current environment will overwrite these during build.
-// The build system defaults to the dev environment which uses `environment.ts`, but if you do
-// `ng build --env=prod` then `environment.prod.ts` will be used instead.
-// The list of which env maps to which file can be found in `.angular-cli.json`.
+// The build system defaults to the dev environment which uses `environment.ts`, but
```

Comment:
The first two lines

```
// The build system defaults to the dev environment which uses `environment.ts`, but
// this file can be replaced during build by using the `fileReplacements` array.
```

Should be replaced with just
```
// This file can be replaced during build by using the `fileReplacements` array.
```

I'm trying to avoid talking about environments, and keep the general ideal to file replacements. This is because the environment doesn't exist at all anymore.

Reply:
Fixed